client/lib/plans/constants.js

```diff
@@ -181,6 +181,14 @@ export const plansList = {
 			FEATURE_GOOGLE_ANALYTICS,
 			FEATURE_NO_BRANDING
 		] ),
+		getPromotedFeatures: () => [
```

Comment:
Do we need/want this? I know that the Business plan has quite a lot of features (and all of them might not fit into a smaller view), but @apeatling mentioned that "Make sure all the latest features for plans are included in these cards"

![selection_138](https://cloud.githubusercontent.com/assets/4988512/17983371/8ba427c6-6b0c-11e6-8bc9-10774fa9103e.png)